tests/generation/test_flax_utils.py

```diff
@@ -290,6 +290,17 @@ def test_beam_search_generate_attn_mask(self):
 
             self.assertListEqual(generation_outputs.tolist(), jit_generation_outputs.tolist())
 
+    def test_greedy_generate_dict_outputs(self):
```

Comment:
Could we expand this test to check that we get the same token ids with `output_scores=False` and `output_scores=True`?

It would be cool if we could also add one slow integration test to a model of your choice (maybe Flax GPT2) where we check that the actual scores we get from the model match the PyTorch equivalent

Reply:
Thanks for iterating on the tests @hannan72 - the second point from my comment above is still pending! Could we check that the actual scores from the Flax models match those from a PyTorch model?